command/agent.go

```diff
@@ -1027,7 +1028,12 @@ func (c *AgentCommand) setBoolFlag(f *FlagSets, configVal bool, fVar *BoolVar) {
 		// Don't do anything as the flag is already set from the command line
 	case flagEnvSet:
 		// Use value from env var
-		*fVar.Target = flagEnvValue != ""
+		val, err := strconv.ParseBool(flagEnvValue)
```

Comment:
What do you think about using the `parseutil.ParseBool` we have in some places within Vault? 

True values:`1, t, T, TRUE, true, True`
False values: `"", 0, f, F, FALSE, false, False`

Reply:
Oh, that seems much better, let me switch to that instead :)

I somehow forgot about our parseutil library when making this :P